interface-definitions/include/pki/dh-parameters.xml.i

```diff
@@ -0,0 +1,14 @@
+<!-- include start from pki/dh-parameters.xml.i -->
```

Comment:
Where is this file referenced? If there are parameters, there should be a completion helper?

Reply:
This file exists for future use, for example here: https://github.com/vyos/vyos-1x/blob/current/interface-definitions/interfaces-openvpn.xml.in#L683

Reply:
Please do not add includes for future use - they will probably not be picked up until you really know it is there. Just send it in a future PR when needed.